xrally_kubernetes/kubernetesclient.py

```diff
@@ -11,12 +11,57 @@
 # License for the specific language governing permissions and limitations
 # under the License.
 
+from kubernetes import client as k8s_config
+from kubernetes.client import api_client
+from kubernetes.client.apis import core_v1_api
+from rally.task import service
 
-class K8sClient(object):
+
+class K8sClient(service.Service):
```

Comment:
Let's rename this class to just ``Kubernetes`` and the whole module to ``service.py``. There is no strong reasoning for that but it would be nice to have naming unifying across all plugins